src/test/resources/correctness/sanity_integration_tests.txt

```diff
@@ -0,0 +1,8 @@
+SELECT AvgTicketPrice, Cancelled, Carrier, FlightDelayMin, timestamp FROM kibana_sample_data_flights
```

Comment:
Curious: are these sanity tests testing basic bug fixes/feature additions we've made recently, or are they just a list of generic queries?

Reply:
Yes, I just added some generic queries. We can add more later.